pages/addhabit/index.js

```diff
@@ -0,0 +1,14 @@
+import AddHabitForm from "@/components /AddHabitsForm";
+import useStore from "@/store";
+
+function AddHabitPage() {
+  const addHabit = useStore((state) => state.addHabit);
+
+  const handleSubmit = (habit) => {
+    addHabit(habit);
+  };
+
+  return <AddHabitForm onSubmit={handleSubmit} />;
+}
```

Comment:
your component does not take any props so you can remove the `addHabit` code here. It is already handled in your `AddHabitForm` component.

```suggestion
function AddHabitPage() {
  return <AddHabitForm />;
}
```